packages/hash/frontend/src/theme/components/inputs/MuiButtonThemeOptions.ts

```diff
@@ -1,11 +1,14 @@
 import { Components, CSSObject, Theme } from "@mui/material";
 
-const buttonFocusBorderOffset = 4;
+const buttonFocusBorderOffset = 6;
 const buttonFocusBorderWidth = 3;
+const baseButtonBorderRadius = 4;
+const largeButtonBorderRadius = 6;
 
 export const MuiButtonThemeOptions: Components<Theme>["MuiButton"] = {
```

Comment:
I added the focus styles (blue outline). Are there some variants the outline doesn't show up in?


https://user-images.githubusercontent.com/16928688/158085994-90e39f6d-092b-4d1e-a517-631523075c56.mov

Reply:
Ah I see, I was expecting to find the styles when using `:focus` (instead of `:focus-visible`). Let's stick to this approach for now, and clarify with @MaggieAppleton what her intentions for the button "focus" styling was when she's back online.